app/scripts/controllers/transactions/index.js

```diff
@@ -430,13 +431,15 @@ export default class TransactionController extends EventEmitter {
     }
 
     if (eip1559Compatibility) {
-      // If the dapp has suggested a gas price, but no maxFeePerGas or maxPriorityFeePerGas
-      //  then we set maxFeePerGas and maxPriorityFeePerGas to the suggested gasPrice.
-      if (
```

Comment:
@NiranjanaBinoy : at this point we should also update `txParams.maxFeePerGas` and `txParams.maxPriorityFeePerGas` with the defaults.